lib/diagnostic.js

```diff
@@ -7,23 +7,43 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(nums)
+{
+  let result;
+  for(let i in nums)
+  {
+    if (result > nums[i]) {result = nums[i];}
+  }
+  return result;
 };
```

Comment:
I think you have the right idea here, but your implementation is not quite right. What's the syntax for for-loop? Please review that. It's very important.